api/src/controllers/transferCredits.ts

```diff
@@ -1,9 +1,37 @@
-import { router } from '../helpers/trpc';
+import { router, publicProcedure } from '../helpers/trpc';
+import { ANTEATER_API_REQUEST_HEADERS } from '../helpers/headers';
+
+interface APExam {
+  fullName: string;
+  catalogueName: string;
+  rewards: {
+    acceptableScores: number[];
+    unitsGranted: number;
+    electiveUnitsGranted: number;
+    geCategories: string[];
+    coursesGranted: {
+      string: string[];
+    };
+  }[];
+}
+
+const getAPExams = async (): Promise<APExam[]> => {
+  const response = await fetch(`${process.env.PUBLIC_API_URL}apExams`, {
+    headers: ANTEATER_API_REQUEST_HEADERS,
+  })
+    .then((res) => res.json())
+    .then((res) => (res.data ? (res.data as APExam[]) : []));
+  return response;
+};
```

Comment:
(subjective thought here)
I personally think this can just go straight in `getAPExamInfo`, as its functionality is single-use and only applies to APs rather than to all of `transferCredits`